apps/protocol-api/src/main.ts

```diff
@@ -535,5 +542,66 @@ app.get('/linear/oauth', async function (req, res) {
   }
 });
 
+app.get('/discord_nonce', async function (req, res) {
+  const nonce = generateNonce();
+  req.session.linearNonce = nonce;
+  res.setHeader('Content-Type', 'text/plain');
+  res.status(200).send(req.session.nonce);
+});
```

Comment:
Why do we need a separate nonce endpoint? Why doesn't the other nonce suffice?

Reply:
we have `nonce` & `linear_nonce`, I thought that we should have nonce for every usage.

Reply:
Ah okay I see, yea it makes sense to have another